packages/remix-server-runtime/jsonify.ts

```diff
@@ -160,14 +167,31 @@ type _tests = [
 
   // object
   Expect<Equal<Pretty<Jsonify<{}>>, {}>>,
+  // @ts-expect-error
+  Expect<Equal<Pretty<Jsonify<{a: any}>>, {a: any}>>,
```

Comment:
I also don't really get why this one is failing tbh 🤔
If I change this to be equal to `{a?: any}`, this passes though